modules/bmo-configuring-userdata.adoc

```diff
@@ -0,0 +1,50 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-configuring-userdata_{context}"]
+= Configuring userData
+
+The cloud image for most operating systems provides the `cloud-init` service, which can read the custom scripts that you create in the `userData` format.
+
+The `userData` field in the `BareMetalHost` custom resource allows you to customize the bare-metal host during provisioning by providing initial configuration data, such as user credentials or system settings. For most operating systems, this is achieved by using a `cloud-init` script encapsulated in a Kubernetes secret.
```

Comment:
```suggestion
Customize the bare-metal host during provisioning by providing initial configuration data in the form of a script by using the `userData` field in the `BareMetalHost` custom resource (CR). For example, you can provide user credentials and system settings. For most operating systems, this is achieved by using a `cloud-init` script encapsulated in a Kubernetes secret. The `cloud-init` service can read the custom scripts that you create in the `userData` format.
```
hm, are we using the field value of a parameter, or creating scripts that run as instructed by the field?